api/src/pcapi/core/offers/api.py

```diff
@@ -55,13 +56,16 @@
 from pcapi.domain import admin_emails
 from pcapi.domain.pro_offers.offers_recap import OffersRecap
 from pcapi.models import db
+from pcapi.models import feature
 from pcapi.models.api_errors import ApiErrors
 from pcapi.models.feature import FeatureToggle
 from pcapi.models.offer_mixin import OfferValidationType
 from pcapi.repository import repository
 from pcapi.repository import transaction
 from pcapi.utils import image_conversion
+from pcapi.utils.cinema_providers import get_boost_film_id_from_uuid
 from pcapi.utils.cinema_providers import get_cds_show_id_from_uuid
+from pcapi.utils.cinema_providers import get_showtime_id_from_uuid
```

Comment:
```suggestion
import pcapi.utils.cinema_providers as cinema_providers_utils
```
Je suis coupable de cet import de fonction pré-existant !